tests/test_http.sh

```diff
@@ -27,19 +27,23 @@ rlJournalStart
     rlPhaseStartTest "Check for SSL warning message"
         rlRun -t -c "docker exec -i kiwi_web /Kiwi/manage.py createsuperuser \
             --username testadmin --email testadmin@domain.com --noinput"
-        rlRun -t -c "docker exec -i kiwi_web /Kiwi/manage.py set_fake_passwords"
+        rlRun -t -c "cat tests/set_testadmin_pass | docker exec -i kiwi_web /Kiwi/manage.py shell"
         IP_ADDRESS=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' kiwi_web`
         rm -f /tmp/testcookies.txt
+        rm -f /tmp/testdata.sslon
         rlRun -t -c "curl -k -L -o- -c /tmp/testcookies.txt https://$IP_ADDRESS:8443/"
         CSRF_TOKEN=`grep csrftoken /tmp/testcookies.txt | cut -f 7`
         rlRun -t -c "curl -e https://$IP_ADDRESS:8443/ -d username=testadmin \
             -d password=password -d csrfmiddlewaretoken=$CSRF_TOKEN -k -L -o /tmp/testdata.sslon \
             -b /tmp/testcookies.txt https://$IP_ADDRESS:8443/accounts/login/"
+        rlAssertGrep "<title>Kiwi TCMS - Dashboard</title>" /tmp/testdata.sslon
         rlAssertNotGrep "You are not using a secure connection." /tmp/testdata.sslon
     rlPhaseEnd
```

Comment:
You can extract the gist of this section and add another rlPhase to check that the warning message exists when using HTTP only. See line 21 for hints.

Reply:
ping ^^^